.github/workflows/ci.yml

```diff
@@ -22,4 +22,5 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: npm ci
     - run: npm link
+    - run: chmod +x ./scripts/testcommands.sh
```

Comment:
afaik git preserves the information whether or not the file is executable or not. Once a file is committed after adding the executable permission 
```
chmod +x testcommands.sh
```

all clones of the repo would have the file marked as executable.

Reply:
Oh, I see. Let me check.